apps/alpha-test-app/src/views/HomeView.vue

```diff
@@ -21,6 +21,7 @@ import {
 } from "sit-onyx";
 import { computed, ref } from "vue";
 import { useRouter } from "vue-router";
+import type { DensityType } from "../../../../packages/sit-onyx/dist/composables/density";
```

Comment:
You can also import the `DENSITY` array from here so we don't need to re-define it here :)
To capitalize the first letter you can import `capitalize` from vue

Reply:
currently we are not exporting the array, I'm trying to export it but it gets removed for some reason and I'm getting errors

Reply:
You probably need to re-start the demo app so it re-builds and recognized the new exported type.
Are you actually using the new array? Otherwise prettier will remove the unused import

Reply:
![image](https://github.com/SchwarzIT/onyx/assets/60856270/50d3d664-c1fd-4e76-b98b-326b8dbc94a4)

Reply:
Maybe the path is not correct?